hardware/i2c.md

```diff
@@ -34,6 +34,7 @@ Overall, this means 0x1D, 0x0E (from v1.3), 0x1E and 0x19 (for the revision) are
 | accessory name | organisation | i2c address(es) used | 
 |----------------|--------------|-----------------------|
 | eg [banana-bit]()|eg Banana enterprises | eg 0x76, 0x29 |
```

Comment:
I think we can just remove the example now we have a real world entry : )
```suggestion
```

Reply:
Sounds good too 👍